GCC warnings

```log
[ 66%] Building C object smtp/CMakeFiles/lib.dir/smtpc.c.o
190 In file included from /tarantool-enterprise/include/tarantool/curl/curl.h:2869:0,
191                  from /tmp/luarocks_smtp-0.0.3-1-6jTe9T/smtp/smtp/smtpc.h:36,
192                  from /tmp/luarocks_smtp-0.0.3-1-6jTe9T/smtp/smtp/smtpc.c:32:
193 /tmp/luarocks_smtp-0.0.3-1-6jTe9T/smtp/smtp/smtpc.c: In function 'smtpc_set_verbose':
194 /tarantool-enterprise/include/tarantool/curl/typecheck-gcc.h:47:39: warning: call to '_curl_easy_setopt_err_long' declared with attribute warning: curl_easy_setopt expects a long argument for this option [enabled by default]
195              _curl_easy_setopt_err_long();                               \
196                                        ^
197 /tmp/luarocks_smtp-0.0.3-1-6jTe9T/smtp/smtp/smtpc.c:134:2: note: in expansion of macro 'curl_easy_setopt'
198   curl_easy_setopt(req->easy, CURLOPT_VERBOSE, curl_verbose);
```

--- a/smtp/lib.c
+++ b/smtp/lib.c
@@ -174,7 +174,7 @@ luaT_smtpc_request(lua_State *L)
 
 	lua_getfield(L, 6, "verbose");
 	if (!lua_isnil(L, -1) && lua_isboolean(L, -1))
-		smtpc_set_verbose(req, true);
+		smtpc_set_verbose(req, lua_toboolean(L, -1));
 	lua_pop(L, 1);
 
 	lua_getfield(L, 6, "username");

--- a/smtp/smtpc.c
+++ b/smtp/smtpc.c
@@ -131,7 +131,7 @@ smtpc_set_body(struct smtpc_request *req, const char *body, size_t size)
 void
 smtpc_set_verbose(struct smtpc_request *req, bool curl_verbose)
 {
-	curl_easy_setopt(req->easy, CURLOPT_VERBOSE, curl_verbose);
+	curl_easy_setopt(req->easy, CURLOPT_VERBOSE, (long)curl_verbose);
 }
 
 void

